pkgs/by-name/or/oreo-cursors-plus/package.nix

```
@@ -0,0 +1,65 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pkgs,
+  cursorsConf ? null, # If set to a string, overwrites contents of './cursors.conf'
+}:
+with lib;
+  stdenv.mkDerivation rec {
+    pname = "oreo-cursors-plus";
+    version = "1.0.20230604";
```

```suggestion
    version = "unstable-2023-06-05";
```
This is the usual naming scheme for untagged versions.